service/kvmconfig/metric/metric.go

```diff
@@ -1,16 +1,15 @@
-package deployment
+package metric
```

Comment:
I had to extract a new package  to avoid cycles. Still `metric.VersionBundleVersionGauge` reads nice.